config.go

```diff
@@ -67,4 +67,6 @@ type State struct {
 	Pid int `json:"pid"`
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
+	// Labels holds runtime-specified information that is not structured by this specification
+	Labels map[string]interface{} `json:"labels"`
```

Comment:
I'd like to reserve the keyword `Labels` for recognized metadata which can be used for grouping.
Can we change this to be `Annotations`?